docs/Writing_easyconfig_files.rst

```diff
@@ -510,12 +510,87 @@ which is equivalent to::
 
   dependencies = [('PnMPI', '1.2.0', '', True)]
 
+
 Using external modules as dependencies
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Since EasyBuild v2.1, specifying modules that are not provided via EasyBuild as dependencies is also supported.
 See :ref:`using_external_modules` for more information.
 
+
+Module Extensions
+~~~~~~~~~~~~~~~~~
+
+Besides dependencies, which are found outside the module being built but are part of the site's EasyBuild installation,
+it is also possible to incorporate extensions to the module within the build.  This is done via the ``exts_list`` array.
+
+Each entry in ``exts_list`` is a three-component tuple, with the name and
+version number, and a dictionary of configuration options for the entry:
+
+.. code:: python
+
+  exts_list = [
+      ('name', 'version', { 'option':'value', 'option':'value' })
+  ]
+
+The latter may contain essentially any of the full EasyConfig parameters, including ``buildopts``, ``installopts``, etc.
+Among those options, the following exceptions and special cases should be noted:
+
+* **nosource**: If set ``True``, no download will be done
```

Comment:
How about
```
***``nosource``**
```
to make it totally clear that these are keys (same for the others below)

Reply:
I was following the same notation used in the bullet list under "Mandatory easyconfig parameters," and elsewhere (in fact, _every_ bullet list of parameters uses bold, but not text).  

If you think we should change them all, I can do that, but all of the parameter lists should be done consistently.  Please reply if you'll like me to change, otherwise I'm going to leave this alone.

Reply:
You're right, consistency is king, this is fine as is